src/Categories/Double/Core.agda

```diff
@@ -0,0 +1,297 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Double.Core where
+
+open import Level
+
+open import Relation.Binary using (Rel; IsEquivalence; Setoid; REL)
+open import Relation.Binary.PropositionalEquality as ≡ using (_≡_)
+import Relation.Binary.Reasoning.Setoid as SetoidR
+
+import Categories.Category.Unbundled as Cat
+
+-- Equality of squares in a double category
+module _ {o ℓ e : Level} {Obj : Set o} (Hor Ver : Cat.Category Obj ℓ e) where
+  private
+    module H = Cat.Category Hor
+    module V = Cat.Category Ver
+    _≈ₕ_ : ∀ {A B} → Rel (A H.⇒ B) e
+    _≈ₕ_ = H._≈_
+    _≈ᵥ_ : ∀ {A B} → Rel (A V.⇒ B) e
+    _≈ᵥ_ = V._≈_
+  record SqEquality
```

Comment:
Some comments here about the purpose and design of this `SqEquality` would be good to add.